### ISSUE TITLE
Example axon data model fits return NaN with default settings

### ISSUE THREAD

**Issue by jtchang** [nicolasC/NNeurite#1]
I'm trying to run the example python notebook on the provided example axon data using Tensorflow 2.9.1 and Python 3.10.

I am using the default learning rate of 0.005, however, when I fit the model, I get NaNs returned. It seems like this may be due to an issue with the Nadam optimizer as altering the learning rate and the number of epochs can bypass this (though results in poor model performance).

**Comment by nicolasC** [comment]
Hi,

you are correct that it must have to do with the optimizer configuration or version. Probably an issue of exploding gradients.

I am myself running the model with tensorflow 2.0.0 and python 3.7.4, without issue.

Those kind of version-dependent issues are not that easy to fix on my side.

I actually have a new version of the model, which I think is more robust, coming up. I need to do a bit of code cleaning beforehand, but I should be able to upload it in the coming days. If you are dissatisfied with the current model, I advise you to wait a bit for the new version to be online and try it.

Thank you for your feedback.
